.github/workflows/cd.yml

```diff
@@ -96,7 +96,12 @@ jobs:
           mask-password: "true"
           registry-type: public
 
-      - name: Push
+      - name: Push primaassicurazioni
+        run: >-
+           buildah manifest push --all localauth0
+           "docker://public.ecr.aws/primaassicurazioni/localauth0:${{ inputs.tag_name || github.event.release.tag_name }}"
```

Comment:
Can you remember me why cannot stay in `primait`? I'm not sure i like this workspace so much

Reply:
The whole company is migrating to this one, primait is not gonna be used for anything in the future. It's out of our hands :smiling_face_with_tear: